client/src/components/MultiJurisdictionAudit/Setup/Review/index.tsx

```diff
@@ -126,6 +93,22 @@ const Review: React.FC<IProps> = ({ prevStage, locked, refresh }: IProps) => {
       j.ballotManifest.processing.status === FileProcessingStatus.PROCESSED
   ).length
 
+  const initialValues: IFormOptions = sampleSizeOptions
+    ? Object.keys(sampleSizeOptions).reduce(
+        (a, contestId) => ({
+          ...a,
+          [contestId]: sampleSizeOptions[contestId][0].size,
+        }),
+        {}
+      )
+    : {}
```

Comment:
what happens if two of the sample size options have the same size number? i think it would break the form.

this isn't a new issue - I think the same issue was there previous to this PR. makes me think that the data structure for sample sizes that the backend gives is not great. each sample size option should have some sort of key or id that you can use to uniquely identify it - the size number is not guaranteed to be uniquely identifying.

thoughts?

Reply:
another potential advantage to that approach would be that you could add in an option with a `custom` key so that you wouldn't need a separate state value to hold the custom size

Reply:
i'm thinking something like this:
```ts
interface ISampleSizeOption {
  key: string
  size: number | null
  prob: number | null
}
```
a few examples:
```ts
{ key: "asn", size: 20, prob: 0.57 }
{ key: "prob90", size: 56, prob: 0.9 }
{ key: "custom", size: null, prob: null } // before the user sets a custom size
{ key: "custom", size: 43, prob: null } // after the user sets a custom size
```

Reply:
@jonahkagan That has merit, yeah. I can picture how that would work on the frontend, and it would definitely clear up that particular bug. Would the `type` field possibly also be marked as 'custom'? It might make things easier on the frontend for generating the details of the different options, but I don't know if that would make it more complex on the backend.

Reply:
I was thinking of getting rid of the `type` field since currently it's only getting used for `asn`. I was thinking you could use the `key` field instead. If you're into this, I'll update my PR with the modified data format

Reply:
@jonahkagan Oh right, I missed that you moved the `asn` to the key. That makes perfect sense. Go for it! :slightly_smiling_face: